keps/sig-scheduling/4664-guarantee-pdb-when-preemption-happens/README.md

```diff
@@ -369,14 +373,14 @@ when selecting victims.
 - if the priority of the preemptor is greater than or equal to the value of `AllowDisruptionByPriorityGreaterThanOrEqual` in victim pod, 
   the implementation will remain consistent with the existing behavior, meaning that the scheduler will try to select victims whose PDBs 
   are not violated by preemption, but if no such victims are found, preemption will still happen and lower priority pods will be preempted 
-  via the `/evictions` endpoint despite their PDBs being violated. If the `/eviction` endpoint returns a response `429 Too Many Requests`
-  and the scheduler will fallback to deletion as an alternative.
+  via the `/evictions` endpoint despite their PDBs being violated. If the `/eviction` endpoint returns a response `429 Too Many Requests`, 
+  the scheduler will fallback to deletion as an alternative.
```

Comment:
I can't remember why this was acceptable. Why not use DELETE directly, given that there is a high chance that the `/evictions` endpoint will just fail?

Reply:
Please take a look at https://github.com/kubernetes/enhancements/pull/4665#discussion_r1628478326. I think we should properly describe the reasoning in the KEP to make it clear for anyone reading it.